### DISCUSSION TITLE
Evangelize .net maui

### DISCUSSION THREAD

**Discussion by codehippie1** [CommunityToolkit/Maui#255] [Q&A] [answered]
Hello fellow mauins,

Context: I started volunteering for .net foundations outreach efforts, also I am the organizer of the largest .net mobile dev meetup group in New York City. Of the few thousand .net mobile devs of the group, only a small portion knows about maui.net

As the saying goes "It doesn't matter if no one knows about it". I am a former Xamarin dev, I haven't heard about maui till I stumble upon a microsoft article few days back.

I would like to find quality speakers on .net maui, to bring the game to nyc mobile devs, tell them maui matter because it does. Help me find quality speakers on .net maui for the largest new york mobile .net meetup.

I can be reached here
LinkedIn: https://www.linkedin.com/in/aneeshlalga/
email: aneeshlalga@gmail.com

**Comment by pictos** [comment, marked as answer]
@codehippie1 I'll be more than happy to give a talk in your meetup, if it's online

**Reply by codehippie1** [reply]
@pictos You are awesome 🥇
1. Would you be able to send in a small speaker bio for us to review(Also we would use that in the event details). 
2. Since there are a ton of topics around maui, what would you like to talk about, A bit of material around that would also help
I would be happy to host an online meetup.  Question - would you be able to let me know if Easter Time 5PM-8PM during weekdays is not an odd time for you?

Since a large meetup community in the greatest city on the planet can only thrive by bringing in speakers frequently, my rest of super hero mauins -where ya at? Don't be hiding, don't make me force to bring in react native speakers. Just kidding. :-)

**Reply by pictos** [reply]
I'll send an email asap

**Comment by CliffAgius** [comment]
Closed as answered.
